internal/process.go

```diff
@@ -181,6 +182,7 @@ func RunExport(args *Args) error {
 		}
 	} else {
 		var exportCreateErr error
+		log.Info().Msg("exporting from folder")
```

Comment:
This message should be more explicity. Like "exporting from local manipulated files folder".